src/modules/song/api.ts

```diff
@@ -450,6 +447,69 @@ export const extendedApi = api.injectEndpoints({
         }
       },
     }),
+    getMintSongPayment: build.query<CborHexResponse, string>({
```

Comment:
I added `getSongMintPayment` in a previous PR recently bc it wasn't documented in the API, we might want to delete that one and keep this one when you merge master